### PR TITLE
UG update with yes, no, start, end and reveal section

### PR DESCRIPTION
**Pull request by WangCheng0116**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -131,46 +131,89 @@ Examples:
 * `find look` returns the flashcard list  and its translation that contains the keyword `look`
 
 
-###  Showing flashcard, starts today’s flashcard session : `start`
+###  Starts review session : `start`
 
-Starts a review session and shows the words the user is going to study sequentially.
+To start a review session, user simply needs to type in `start` command, and a new review session will be started.
```
Comment (A1WAYSD) [inline docs/UserGuide.md:136]: Feels a bit wordy here since "start a review session" is repeated twice. Will it be better to reorganize the sentence?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -131,46 +131,89 @@ Examples:
...
 Format: `reveal`
 
-Output :
-![img.png](images/Reveal.png)
+Output : `The translation is [CURRENT FLASHCARD'S TRANSLATION]`  
+
+Examples:![img.png](images/Reveal.png)
+
+**Note**
+* `reveal` command will only take effect during review session. Otherwise, error message`You are not in a review session.`
```
Comment (A1WAYSD) [inline docs/UserGuide.md:173]: Would be better to add a white space between "error message" and "You".

File: docs/UserGuide.md [hunk 3]
```diff
@@ -131,46 +131,89 @@ Examples:
...
 
 Format: `yes`
 
+Output: ![img.png](images/Yes.png)
+or
+![img.png](images/Yes2.png)
+if there's no word left in the review session.
+
+**Note**
+* `yes` command will only take effect during review session. Otherwise, error message`You are not in a review session.`
```
Comment (A1WAYSD) [inline docs/UserGuide.md:190]: Similar to above.

**Review by A1WAYSD — COMMENTED** [review]
LGTM!

**Comment by A1WAYSD** [comment]
Perhaps it would be better for our whole group to use "flash card" instead of "flashcard" in documentation. Since our classes are all named "FlashCard"!

File: docs/UserGuide.md (hunk 3 above)
Comment (WangCheng0116) [reply docs/UserGuide.md]: I have changed accordingly.

File: docs/images/No.png
Comment (itsNatTan) [inline docs/images/No.png:1]: Maybe can reduce the whitespace at the bottom.

**Review by itsNatTan — APPROVED** [review]
Other than the whitespace, LGTM
